Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -10443,8 +10445,15 @@ void WebPageProxy::requestAttachmentIcon(const String& identifier, const String&
     }
 #endif
 
+    String exampleFileName;
+    size_t dotIndex = fileName.reverseFind('.');
+    if (dotIndex != notFound) {
+        String extension = fileName.substring(dotIndex);
```

Comment:
We don't need to construct a temporary String here:
`auto extension = StringView(fileName).substring(dotIndex);`